qiskit/ignis/experiments/calibration/methods/basic_1q.py

```diff
@@ -92,35 +97,60 @@ def __init__(self,
             qubit: Qubit on which to run the calibration.
             data_processing: Steps used to process the data from the Result.
             amp_vals: Amplitude values to scan in the calibration.
-            analysis_class: Analysis class used.
-            job: Optional job id to retrieve past experiments.
-            pulse_name: Pulse name in the database entry to provide parameter set to
+            gate_name: Pulse name in the database entry to provide parameter set to
                 construct pulse schedule to calibrate. By default pi pulse parameter is used.
+            update_pulses: A dict of pulses and their rotation angle to update in the pulse
```

Comment:
is it possible to update x90p of cr gate if it is independently defined?

Reply:
done in 5b4eaa9